<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBar</class>
 <widget class="QFrame" name="SearchBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="searchButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/resources/icons/actions/edit-find.png</normaloff>:/resources/icons/actions/edit-find.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Matchcase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="text">
      <string notr="true">3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultSeparatorLabel">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultCountLabel">
     <property name="text">
      <string notr="true">5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextResultButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/resources/icons/actions/go-down.png</normaloff>:/resources/icons/actions/go-down.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousResultButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/resources/icons/actions/go-up.png</normaloff>:/resources/icons/actions/go-up.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="firstResultButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/resources/icons/actions/go-top.png</normaloff>:/resources/icons/actions/go-top.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lastResultButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/resources/icons/actions/go-bottom.png</normaloff>:/resources/icons/actions/go-bottom.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
